Kconfig

```diff
@@ -974,6 +974,18 @@ menu "LVGL configuration"
         config LV_USE_MONKEY
             bool "Enable Monkey test"
             default n
+
+        config LV_USE_GRIDNAV
+            bool "Enable grid navigation"
+            default n 
+
+        config LV_USE_FRAGMENT
+            bool "Enable lv_obj fragment"
+            default n 
+
+        config LV_USE_IMGFONT
```

Comment:
This is not part of upstream commit, I think you added it accidentally when cherry-picking.